lib/Frontend/nvqpp/ConvertExpr.cpp

```diff
@@ -1415,6 +1416,13 @@ bool QuakeBridgeVisitor::VisitCallExpr(clang::CallExpr *x) {
           processedArgs.push_back(v);
         } else if (auto load = v.getDefiningOp<cudaq::cc::LoadOp>()) {
           processedArgs.push_back(load.getPtrvalue());
+        } else if (auto computePtr =
+                       v.getDefiningOp<cudaq::cc::ComputePtrOp>()) {
+          auto ptrEleTy = computePtr.getType().getElementType();
+          Value loaded = builder.create<cudaq::cc::LoadOp>(loc, ptrEleTy, v);
```

Comment:
This load doesn't look right. The other cases are passing the pointer to the Pauli string data. Shouldn't this just forward the pointer computed by `ComputePtr`?